packages/api/src/trpc.ts

```diff
@@ -52,3 +55,38 @@ export const protectedProcedure = t.procedure.use(({ ctx, next }) => {
     },
   });
 });
+
+export const calendarProcedure = protectedProcedure.use(
+  async ({ ctx, next }) => {
+    const allAccounts = await getAllAccounts(ctx.user, ctx.headers);
+
+    const activeAccount = allAccounts.find(
+      (account) => account.id === ctx.user.defaultAccountId,
+    );
+
+    if (!activeAccount) {
+      throw new TRPCError({
+        code: "PRECONDITION_FAILED",
+        message: "No active account set",
+      });
+    }
```

Comment:
The only thing I see when I run dev. 

You're filtering out other errors in `getAllAccounts`.
Consequently, we only discover something is wrong here, and now we don't know what they were and how to handle them.